test/js/node/tls/node-tls-server.test.ts

```diff
@@ -340,6 +341,69 @@ describe("tls.createServer events", () => {
     );
   });
 
+  it("should work with getCertificate", done => {
+    let timeout: Timer;
+    let client: TLSSocket | null = null;
+    const server: Server = createServer(COMMON_CERT, socket => {
+      socket.on("secure", () => {
+        try {
+          expect(socket).toBeDefined();
+          const cert = socket.getCertificate() as PeerCertificate;
+          expect(cert).toBeDefined();
+          expect(cert.subject).toBeDefined();
+          expect(cert.subject).toMatchObject({
+            C: "AU",
+            ST: "Some-State",
+            O: "Internet Widgits Pty Ltd",
+          });
+
+          expect(cert.issuer).toBeDefined();
+          expect(cert.issuer).toMatchObject({
+            C: "AU",
+            ST: "Some-State",
+            O: "Internet Widgits Pty Ltd",
+          });
+
+          expect(cert.ca).toBeTrue();
+          expect(cert.bits).toBeNumber();
+          expect(typeof cert.modulus).toBe("string");
+          expect(typeof cert.exponent).toBe("string");
+          expect(cert.pubkey).toBeInstanceOf(Buffer);
+          expect(typeof cert.valid_from).toBe("string");
+          expect(typeof cert.valid_to).toBe("string");
```

Comment:
Can these tests assert a specific value and check its the same in Node & Bun? Would be easy for that code that encodes as either latin1 or utf-8 to have some subtle bug we never catch because we aren't checking the specific values

Reply:
fix on https://github.com/oven-sh/bun/commit/927f54269bdf489d21be4a8a6b89b448fdf48184